src/parser/name_resolution.c

```diff
@@ -6995,6 +6995,31 @@ pt_resolve_hint (PARSER_CONTEXT * parser, PT_NODE * node)
   return ER_FAILED;
 }
 
+static void
+pt_write_warning (PARSER_CONTEXT * parser, PT_NODE * index, int lineNo, int erSetNo, int msgNo)
```

Comment:
code convention: no camal case
```suggestion
pt_write_warning (PARSER_CONTEXT * parser, PT_NODE * index, int line_no, int er_set_no, int msg_no)
```

Reply:
I will fix it, thanks.